monkey/tests/unit_tests/monkey_island/cc/resources/test_island_mode.py

```diff
@@ -13,7 +13,11 @@ def uses_database():
 
 
 @pytest.mark.parametrize("mode", ["ransomware", "advanced"])
-def test_island_mode_post(flask_client, mode):
+def test_island_mode_post(flask_client, mode, monkeypatch):
```

Comment:
Shouldn't we also check if config is changed? Maybe we should do it somewhere else, but we should probably have UT's for applying filters/templates to config